### ISSUE TITLE
[Suggestion] Allow Furniture assets to be any item (Not just "OAK_WOOD")

### ISSUE THREAD

**Issue by ArgonGaming** [PluginBugs/Issues-ItemsAdder#662]
**Is your feature request related to a problem? Please describe.**
It may be related to a limitation problem. Meaning this is a feature that is practically going to allow many more options. Currently you must set a furniture item to be the specific block "OAK_WOOD". I see no reason to this, when it is easily as easy to simply use the Custom Item IDs to achieve the same thing.

**Describe the solution you'd like**
Why can we not set the item as paper, or a potion? Then we simply readjust what the "head display" is on a model. It works when I put it on my characters head, therefor it should work on armor stands (or item frames if you end up adding that as a method) just as well.

This allows for multi-purpose items. For example, the tea cup I have. I set it to a potion, and made it so players can drink the tea. It would be nice to be able to place it as well as an armor stand, with a potion on its head technically. Either way, even if it was set as paper, it would be nice to not have to set it to a block just to place furniture!

**Describe alternatives you've considered**
Not sure if there are any alternatives, this is a very specific function

**Additional context**
![image](https://user-images.githubusercontent.com/30943826/107729282-693b1d80-6cbe-11eb-9bca-70ed8bff507a.png)
![image](https://user-images.githubusercontent.com/30943826/107729303-77893980-6cbe-11eb-87b5-9be418380ef2.png)

**Comment by LoneDev6** [comment]
Furniture place event is called only by materials who can be placed in vanilla (which trigger the blockplace event), this cannot be changed as it would surely generate incompatibilities with protection plugins and similar.

Other than this I found another issue in the idea: the interact event is called when the user drinks tea while looking at a block, so if I allow somehow to place furnitures which use the potion material it will get placed right after user starts drinking it which is annoying.

I want to clarify that the idea is very cool, I like it but I'm sure it will cause a lot of problems if allowed.

You (or a friend) may implement it as a skript/Java addon btw, but keep in mind that it may cause the issues I stated above.


Let me know if you have any idea on how to fix the states issues, I may implement that in itemsadder directly
